js/lbank.js

```diff
@@ -585,6 +585,10 @@ module.exports = class lbank extends Exchange {
     }
 
     async fetchClosedOrders (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        if (symbol !== undefined) {
+            const market = this.market (symbol);
```

Comment:
We must ensure `await this.loadMarkets ()` in this case, cause otherwise a call to `this.market()` will fail.